.github/workflows/pr.yml

```diff
@@ -30,16 +41,85 @@ jobs:
             ${{ runner.os }}-yarn-
 
       - name: Install dependencies
-        run: yarn install --frozen-lockfile
+        run: yarn install --frozen-lockfile --ignore-engines
 
       - name: Build packages
         run: yarn build
 
-      # - name: Build CRA templates
-      #   run: yarn build:templates ../cra-templates
-
       - name: Lint types and code
         run: yarn lint
 
       - name: Run tests
         run: yarn test:ci
+
+  release:
+    name: Release Snapshot (if labelled)
+    runs-on: ubuntu-latest
+    if: contains( github.event.pull_request.labels.*.name, 'release:canary' )
+    steps:
+      - name: Cancel previous runs
```

Comment:
I would be very careful with that. This can cause quite strange side effects, depending on the time at which a run is aborted.

I would rather make sure that only one job running at any time, and the next one is queued.